data/adobe-activation

```diff
@@ -1,139 +1,1264 @@
-full:wwis-dubc1-vip120.adobe.com
-full:wwis-dubc1-vip121.adobe.com
-full:wwis-dubc1-vip122.adobe.com
-full:wwis-dubc1-vip123.adobe.com
-full:wwis-dubc1-vip124.adobe.com
-full:wwis-dubc1-vip125.adobe.com
-full:wwis-dubc1-vip30.adobe.com
-full:wwis-dubc1-vip31.adobe.com
-full:wwis-dubc1-vip32.adobe.com
-full:wwis-dubc1-vip33.adobe.com
-full:wwis-dubc1-vip34.adobe.com
-full:wwis-dubc1-vip35.adobe.com
-full:wwis-dubc1-vip36.adobe.com
-full:wwis-dubc1-vip37.adobe.com
-full:wwis-dubc1-vip38.adobe.com
-full:wwis-dubc1-vip39.adobe.com
-full:wwis-dubc1-vip40.adobe.com
-full:wwis-dubc1-vip41.adobe.com
-full:wwis-dubc1-vip42.adobe.com
-full:wwis-dubc1-vip43.adobe.com
-full:wwis-dubc1-vip44.adobe.com
-full:wwis-dubc1-vip45.adobe.com
-full:wwis-dubc1-vip46.adobe.com
-full:wwis-dubc1-vip47.adobe.com
-full:wwis-dubc1-vip48.adobe.com
-full:wwis-dubc1-vip49.adobe.com
-full:wwis-dubc1-vip50.adobe.com
-full:wwis-dubc1-vip51.adobe.com
-full:wwis-dubc1-vip52.adobe.com
-full:wwis-dubc1-vip53.adobe.com
-full:wwis-dubc1-vip54.adobe.com
-full:wwis-dubc1-vip55.adobe.com
-full:wwis-dubc1-vip56.adobe.com
-full:wwis-dubc1-vip57.adobe.com
-full:wwis-dubc1-vip58.adobe.com
-full:wwis-dubc1-vip59.adobe.com
-full:wwis-dubc1-vip60.adobe.com
-full:wwis-dubc1-vip61.adobe.com
-full:wwis-dubc1-vip62.adobe.com
-full:wwis-dubc1-vip63.adobe.com
-full:wwis-dubc1-vip64.adobe.com
-full:wwis-dubc1-vip65.adobe.com
-full:wwis-dubc1-vip66.adobe.com
-full:wwis-dubc1-vip67.adobe.com
-full:wwis-dubc1-vip68.adobe.com
-full:wwis-dubc1-vip69.adobe.com
-full:wwis-dubc1-vip70.adobe.com
-full:wwis-dubc1-vip71.adobe.com
-full:wwis-dubc1-vip72.adobe.com
-full:wwis-dubc1-vip73.adobe.com
-full:wwis-dubc1-vip74.adobe.com
-full:wwis-dubc1-vip75.adobe.com
-full:wwis-dubc1-vip76.adobe.com
-full:wwis-dubc1-vip77.adobe.com
-full:wwis-dubc1-vip78.adobe.com
-full:wwis-dubc1-vip79.adobe.com
-full:wwis-dubc1-vip80.adobe.com
-full:wwis-dubc1-vip81.adobe.com
-full:wwis-dubc1-vip82.adobe.com
-full:wwis-dubc1-vip83.adobe.com
-full:wwis-dubc1-vip84.adobe.com
-full:wwis-dubc1-vip85.adobe.com
-full:wwis-dubc1-vip86.adobe.com
-full:wwis-dubc1-vip87.adobe.com
-full:wwis-dubc1-vip88.adobe.com
-full:wwis-dubc1-vip89.adobe.com
-full:wwis-dubc1-vip90.adobe.com
-full:wwis-dubc1-vip91.adobe.com
-full:wwis-dubc1-vip92.adobe.com
-full:wwis-dubc1-vip93.adobe.com
-full:wwis-dubc1-vip94.adobe.com
-full:wwis-dubc1-vip95.adobe.com
-full:wwis-dubc1-vip96.adobe.com
-full:wwis-dubc1-vip97.adobe.com
-full:wwis-dubc1-vip98.adobe.com
-full:wwis-dubc1-vip99.adobe.com
+# These IPs will only block the telemetry check of Adobe apps,
+# and shouldn't interfere with any other network option,
+# such as AI generation, downloads, etc.
+# 0.0.0.0 can be found in https://a.dove.isdumb.one/list.txt
+# Pop up window? Report at:
+# https://github.com/ignaciocastro/a-dove-is-dumb/issues
+# Or follow updates at https://a.dove.isdumb.one/telegram
+# Last update: 2025-02-02 22:54 UTC
+ic.adobe.io
+cc-api-data.adobe.io
+0mo5a70cqa.adobe.io
+1b9khekel6.adobe.io
```

Comment:
It makes no point in abusing the `domain:` suffix, these domains should be matched with the `full:` suffix.